packages/utils/test/schema/getDefaultFormStateTest.ts

```diff
@@ -1497,6 +1693,30 @@ export default function getDefaultFormStateTest(testValidator: TestValidatorType
           )
         ).toEqual(['ConstFromRoot', 'ConstFromRoot', 'Constant', 'Constant']);
       });
+      it('test an array const value NOT populate as defaults when constAsDefaults is never', () => {
+        const schema: RJSFSchema = {
+          type: 'array',
+          minItems: 4,
+          const: ['ConstFromRoot', 'ConstFromRoot'],
+          items: {
+            type: 'string',
+            const: 'Constant',
+          },
+        };
+
+        expect(
+          getArrayDefaults(
+            testValidator,
+            schema,
+            {
+              rootSchema: schema,
+              includeUndefinedValues: 'excludeObjectChildren',
+              experimental_defaultFormStateBehavior: { constAsDefaults: 'never' },
+            },
+            ['ConstFromRoot', 'ConstFromRoot']
```

Comment:
kind of confusing that `defaults` are passed but not captured in the schema, but this should be fine